lib/builder/components/ShareDashboard.js

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect } from "react";
+import { connect } from "react-redux";
+import { withRouter } from "react-router-dom";
+import {
+  toogleDashboardsMenu,
+  setAccessKey,
+  clearAccessKey,
+} from "../../actions/rootActions";
+import client from "../../../config/analysis";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const ShareDashboards = props => {
+  useEffect(() => {
+    const getKeyFromAPI = async () => {
+      const keyName = `public-dashboard:${props.dashboardTitle}`;
```

Comment:
let's add a space after `:` to make it more readable for people. it'll be automatically url-encoded anyway during requests